src/main/java/uk/gov/hmcts/divorce/divorcecase/NoFaultDivorce.java

```diff
@@ -29,6 +30,7 @@ public void configure(final ConfigBuilder<CaseData, State, UserRole> configBuild
 
         configBuilder.caseType(CASE_TYPE, CASE_TYPE, "Handling of the dissolution of marriage");
         configBuilder.jurisdiction(JURISDICTION, "Family Divorce", "Family Divorce: dissolution of marriage");
+        configBuilder.omitHistoryForRoles(SOLICITOR, APPLICANT_2_SOLICITOR);
```

Comment:
I assume both app1 and app2 solicitor will always have SOLICITOR role ? so should we only add SOLICITOR ?

Reply:
The case will also have [APPTWOSOLICITOR] role so caseHistory field will be accessible.

Reply:
ahh got you :)